frontend/apps/thunder-develop/src/features/flows/components/resource-property-panel/rich-text/helper-plugins/__tests__/CustomLinkPluginPredefinedUrls.test.tsx

```
@@ -67,18 +67,23 @@ vi.mock('react-i18next', () => ({
+      getEditorState: mockGetEditorState,
+    },
+  ],
 }));
 
 // Mock lexical utils
 vi.mock('@lexical/utils', () => ({
-  mergeRegister: (...fns: (() => void)[]) => () => fns.forEach(fn => fn()),
+  mergeRegister:
+    (...fns: (() => void)[]) =>
+    () =>
+      fns.forEach((fn) => fn()),
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)